platforms/f2/setup-f2-ioc.ts

```diff
@@ -0,0 +1,16 @@
+import { BaseContextSetup } from '@platforms/shared';
+import { context, InstantConfigService } from '@wikia/ad-engine';
+import { Container } from '@wikia/dependency-injection';
+import { set } from 'lodash';
+import * as fallbackInstantConfig from './fallback-config.json';
+import { F2BaseContextSetup } from './setup/f2-base-context.setup';
+
+export async function setupF2Ioc(): Promise<Container> {
+	const container = new Container();
+
+	set(window, context.get('services.instantConfig.fallbackConfigKey'), fallbackInstantConfig);
+	container.bind(InstantConfigService as any).value(await InstantConfigService.init());
```

Comment:
I think this is no longer necessary.
```suggestion
	container.bind(InstantConfigService).value(await InstantConfigService.init());
```

Reply:
Why you think so? What has changed?

Reply:
In the early version of DI lib, there was problem with types, that caused us to add this `as any`. It has been resolved for some time now.